Text/Distinct/README.md

```diff
@@ -0,0 +1,100 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-search
+name: Distinct sample skill for cognitive search
+description: This custom skill removes duplicates from a list of terms.
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Text/Distinct/azuredeploy.json
+---
+
+# Distinct
+
+This custom skill removes duplicates from a list of terms.
+
```

Comment:
include that the skill is also case insensitive and ignores punctuation and seperator characters